db/migrate/20230730151848_create_cohorts.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCohorts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :cohorts do |t|
+      t.text :name, null: false
+      t.references :school, null: false, foreign_key: true
+      t.references :school_year, null: false, foreign_key: true
+      t.integer :grade, null: false
```

Comment:
Just to not confuse this with grades of A,B,C,D,F, etc, could we call this field "grade_level"?